.gitignore

```diff
@@ -24,3 +24,4 @@ config.log*
 /node_modules/
 /bindings/nodejs/generated/main.cc
 /bindings/nodejs/generated/main.h
+/bindings/python/generated/mainloop.c
```

Comment:
maybe ignore all inside /bindings/python/generated?

also, try to build these generated inside /build? like `/build/stage/python`?